macros/compare_column_values.sql

```diff
@@ -17,13 +17,13 @@ joined as (
         a_query.{{ column_to_compare }} as a_query_value,
         b_query.{{ column_to_compare }} as b_query_value,
         case
-            when a_query.{{ column_to_compare }} = b_query.{{ column_to_compare }} then '✅: perfect match'
-            when a_query.{{ column_to_compare }} is null and b_query.{{ column_to_compare }} is null then '✅: both are null'
-            when a_query.{{ primary_key }} is null then '🤷: ‍missing from a'
-            when b_query.{{ primary_key }} is null then '🤷: missing from b'
-            when a_query.{{ column_to_compare }} is null then '🤷: value is null in a only'
-            when b_query.{{ column_to_compare }} is null then '🤷: value is null in b only'
-            when a_query.{{ column_to_compare }} != b_query.{{ column_to_compare }} then '🙅: ‍values do not match'
+            when a_query.{{ column_to_compare }} = b_query.{{ column_to_compare }} then '{% if emojis %}✅: {% endif %}perfect match'
+            when a_query.{{ column_to_compare }} is null and b_query.{{ column_to_compare }} is null then '{% if emojis %}✅: {% endif %}both are null'
+            when a_query.{{ primary_key }} is null then '{% if emojis %}🤷: {% endif %}missing from {{ a_relation_name }}'
+            when b_query.{{ primary_key }} is null then '{% if emojis %}🤷: {% endif %}missing from {{ b_relation_name }}'
+            when a_query.{{ column_to_compare }} is null then '{% if emojis %}🤷: {% endif %}value is null in {{ a_relation_name }} only'
+            when b_query.{{ column_to_compare }} is null then '{% if emojis %}🤷: {% endif %}value is null in {{ b_relation_name }} only'
+            when a_query.{{ column_to_compare }} != b_query.{{ column_to_compare }} then '{% if emojis %}🙅: {% endif %}‍values do not match'
```

Comment:
```suggestion
            when a_query.{{ column_to_compare }} != b_query.{{ column_to_compare }} then '{% if emojis %}❌: {% endif %}‍values do not match'
```

🤷 and 🙅 look unnecessarily similar given that we have the entire emoji space available to us